internal/version/build.go

```diff
@@ -21,6 +23,7 @@ var platform = fmt.Sprintf("%s/%s", runtime.GOOS, runtime.GOARCH)
 // Version defines the application version details (generally from build information)
 type Version struct {
 	Version        string `json:"version"`        // application semantic version
+	SchemaVersion  string `json:"schemaVersion"`  // application semantic JSON schema version
```

Comment:
nit: maybe `jsonSchemaVersion` in case we end up adding schemas for other formats in the future?